pkgs/by-name/ve/versionCheckHook/hook.sh

```diff
@@ -1,6 +1,12 @@
 _handleCmdOutput(){
     local versionOutput
-    versionOutput="$(env --chdir=/ --argv0="$(basename "$1")" --ignore-environment "$@" 2>&1 || true)"
+    versionOutput="$(env \
+        --chdir=/ \
+        --argv0="$(basename "$1")" \
+        --ignore-environment \
+        "$@" 2>&1 \
+        | sed -e 's|@storeDir@/[^/]*/|{{storeDir}}/|g' \
```

Comment:
```suggestion
        | sed -e 's|@storeDir@/[^/ ]*/|{{storeDir}}/|g' \
```

I added the same tweak to mine

Reply:
> I added the same tweak to mine

Done.